src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -170,6 +170,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ResolvePackageDependencies
       ProjectPath="$(MSBuildProjectFullPath)"
       ProjectAssetsFile="$(ProjectAssetsFile)"
+      References="@(Reference)"
```

Comment:
>References="@(Reference)" [](start = 6, length = 25)

Are we using this approach because the list of references is always strictly smaller than the list of DLLs listed in the assets file under NETStandard.Library? (Otherwise we could just mark all those assemblies as dependencies and avoid this parameter). Also, what ordering requirement guarantees that ResolvePackageDependencies is always called after the target the sets the Netstandard library references?